tests/Rules/Fixture/laravel-mailable-method.php

```diff
@@ -17,4 +18,17 @@ public function build()
             'foo' => 'bar',
         ]);
     }
+
+    /**
+     * Get the message content definition.
+     */
+    public function content(): Content
```

Comment:
It would probably be best to create a separate test class for this one so that it doesn't get obscured the other call triggering the same warnings.